it/lessons/basics/collections.md

```diff
@@ -46,7 +51,20 @@ iex> ["foo", :bar, 42] -- [42, "bar"]
 ["foo", :bar]
 ```
 
-**Nota:** viene usata la [strict comparison](../basics#confronto) per controllare i valori.
+Fate attenzione a valori duplucati. Per ogni elemento nella lista di destra, solo la prima occorrenza nella lista di sinistra viene rimossa.
```

Comment:
Typo here: `duplucati` should be `duplicati`

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b